Programmers/Level2/YJ_49993.java

```diff
@@ -0,0 +1,38 @@
+import java.util.*;
+
+public class YJ_49993 {
+    public static void main(String[] args) {
+        String skill = "CBD";
+        String[] skill_trees = {"BACDE", "CBADF", "AECB", "BDA"};
+
+        int answer = 0;
+        Map<Character,Integer> skillMap = new HashMap<>();
+        for(int i=0; i<skill.length(); i++){
+            skillMap.put(skill.charAt(i),i+1);
+        }
+
+        for(String tree :skill_trees){
+            boolean temp = true;
+            int order = 1;
+
+            for(int i=0; i<tree.length(); i++){
+                Integer now = skillMap.get(tree.charAt(i));
+                if(now == null){
+                    continue;
+                }
```

Comment:
```suggestion
                if (!skillMap.containsKey(tree.charAt(i))) {
                    continue;
                }
                Integer now = skillMap.get(tree.charAt(i));
```
저는 null 처리보다는 미리 값이 존재하는지 체크해주는 것을 좋아하는데 한번 적어봤어요!

Reply:
값 존재유무를 `containsKey`로 정확히 명시해주니까 코드 가독성이 좋네요!